src/modules/SearchPage/components/Pagination/index.tsx

```diff
@@ -0,0 +1,41 @@
+import { usePaginationRange } from 'hooks';
+import { DOTS } from 'appConstants';
+import { MouseEvent } from 'react';
+import * as S from './styled';
+
+type Props = {
+  currentPage: number;
+  totalPageCount: number;
+  nextPage: () => void;
+  previousPage: () => void;
+  specificPage: (event: MouseEvent<HTMLButtonElement>) => void;
+};
+export const Pagination = (props: Props) => {
+  const { totalPageCount, currentPage, nextPage, previousPage, specificPage } = props;
+  const paginationRange = usePaginationRange({
+    totalPageCount,
+    currentPage,
+  });
+
+  return (
+    <S.Pagination>
+      <S.PaginationButton onClick={previousPage} disabled={currentPage === 1}>
+        👈
+      </S.PaginationButton>
+
+      {paginationRange?.map((item, index) =>
```

Comment:
so you will not need if there
```suggestion
      {paginationRange.map((item, index) =>
```

Reply:
Actually in usePaginationRange there was missing a "else return []" statement. After changing that variable paginationRange will always be an array and questionmartk (paginationRange?.map) will not be necessary.